### ISSUE TITLE
Solution for the busted snapshots

### ISSUE THREAD

**Issue by blxdyx** [node-real/bsc-erigon#234]
#### Description
The history data in the ./data/snapshots meet some error.

#### How to fix
1. Check the latest commit [24b2aaa](https://github.com/node-real/bsc-erigon/commit/24b2aaa6f5292cd892374f371e71744c04711521).
2. make erigon && make integration
3. cd ./data/snapshots and remove all the snapshots after block 28,500,000, such as 'v1-0285000-029000*' and  also remove the 'db' in this dir. (If dont work, may delete all the snapshots)
4. integration stage_headers --reset
5. run with --downloader.verify

Or just download the latest snapshot and use the latest commit resync(Spend more time).

#### Expected behaviour
After finish the steps, it will fix the busted snapshots.

**Comment by rayn316** [comment]
An error occurred while executing this command.
cmd: `./bin/integration stage_headers --reset --datadir=/node/archive/bsc/erigon/ --chain=bsc `
```
INFO[10-23|11:29:17.565] Clear                                    table=MaxTxNum
INFO[10-23|11:29:37.569] [fillind_db_from_snapshots] Total difficulty index: 2756k/27999k
INFO[10-23|11:29:57.569] [fillind_db_from_snapshots] Total difficulty index: 5013k/27999k
INFO[10-23|11:30:17.568] [fillind_db_from_snapshots] Total difficulty index: 8117k/27999k
INFO[10-23|11:30:37.569] [fillind_db_from_snapshots] Total difficulty index: 11197k/27999k
INFO[10-23|11:31:00.853] [fillind_db_from_snapshots] Total difficulty index: 14913k/27999k
INFO[10-23|11:31:17.568] [fillind_db_from_snapshots] Total difficulty index: 18250k/27999k
INFO[10-23|11:31:37.568] [fillind_db_from_snapshots] Total difficulty index: 21393k/27999k
INFO[10-23|11:31:57.568] [fillind_db_from_snapshots] Total difficulty index: 24417k/27999k
INFO[10-23|11:32:17.569] [fillind_db_from_snapshots] Total difficulty index: 27504k/27999k
EROR[10-23|11:32:21.004] unknown ancestor
```

**Comment by blockPI-rayn** [comment]
@blxdyx yes, The tools are all the latest commit.
erigon version 1.1.9-dev-24b2aaa6

**Comment by rayn316** [comment]
@blxdyx yes, The tools are all the latest commit.
erigon version 1.1.9-dev-24b2aaa6

**Comment by tpalaz** [comment]
I can confirm this is working as expected. My node was stuck on block 29.9 million for quite some time from a scratch sync. 

I followed the instructions exactly, but decided to go back to block ~29.5 and sync from there as I had messed around with resetting headers a few times and some other things.

Incase anyone needs  this, here's a useful command to find the snapshot files above 29.5 million (can also pipe to exec rm {} \ to delete):

`find . -type f -regextype posix-extended -regex ".*-0(29[5-9][0-9]{2}|[3-9][0-9]{4})-.*"`
